mmv1/products/alloydb/terraform.yaml

```diff
@@ -76,11 +76,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           alloydb_instance_name: "alloydb-instance"
           network_name: "alloydb-network"
         test_vars_overrides:
-          network_name: 'BootstrapSharedTestNetwork(t, "alloydb")'
+          network_name: 'BootstrapSharedTestNetwork(t, "alloydb-basic")'
```

Comment:
stray thought: I wonder if we could derive a name from `t` rather than require a unique one be supplied by the author. Would guard against copy-paste issues.

Reply:
That's a great idea! Agreed that copy-paste is going to be an issue moving forward.